src/lib/location/countries.ts

```diff
@@ -5,7 +5,7 @@
  * ]
  * */
 export const aliases = [
-	[ 'gb', 'uk', 'united kingdom', 'england', 'northern ireland', 'wales', 'scotland' ],
+	[ 'gb', 'uk', 'great britain', 'england', 'northern ireland', 'wales', 'scotland' ],
```

Comment:
Shouldn't `united kingdom` still be in the list?

Reply:
> United Kingdom is the official name, so we already have that in the index.